docker/Dockerfile.https

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:20.04
+
+# Get arguments from the build command line
+ARG version \
+    keystore_pass \
+    key_pass
+
+ENV VERSION=$version
+
+# Build up the OS
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y curl \
+                       libtcnative-1 \
+                       maven \
+                       openjdk-11-jdk-headless \
+                       tar
+
+# Make room for the app and capture the version
+RUN mkdir /usr/local/registry-api-service \
+ && cd /usr/local/registry-api-service \
+ && echo ${VERSION} > version.txt
+
+# Copy the data into the building container
+COPY LICENSE.txt /usr/local/registry-api-service
+COPY NOTICE.txt /usr/local/registry-api-service
+COPY README.txt /usr/local/registry-api-service
+
+# Resources shared with the rest of the world
+EXPOSE 443
+
+# Obtain the release jar file 
+RUN set -ex \
+ && cd /usr/local/registry-api-service \
+ && curl -L \
+         https://github.com/NASA-PDS/registry-api-service/releases/download/v${VERSION}/registry-api-service-${VERSION}.jar \
+         -o registry-api-service.jar
+
+# TODO: Set-up bind dir for application.properties here...
+
+# Now generate/store the self signed cert.
+RUN cd /usr/local/registry-api-service \
+ && keytool -genkeypair -keyalg RSA -alias registry -keystore keystore.p12 -storepass $keystore_pass -keypass $key_pass \
+            -dname "CN=Unknown" -keysize 2048 -validity 3650 -noprompt 
```

Comment:
I'm not sure why we'd ever need a container that spoke `https`. Wouldn't SSL/TLS handling be the job of the ALB or Apache or Nginx that runs in front of this? Help me out here 😇

Reply:
For now, the SA's have said it's OK to use http but we should be prepared for the time the Infosec guys mandate that all connections must be SSL. We can get away with it until then because we're not passing credentials or anything that should otherwise be secure. We'll leave the https versions here but consider them on the back shelf.